Kernel/Library/LockRefPtr.h

```diff
@@ -10,6 +10,7 @@
 #include <AK/Atomic.h>
 #include <AK/Error.h>
 #include <AK/Format.h>
+#include <AK/OwnPtr.h>
```

Comment:
Why do we have to include OwnPtr here? Does forward not work? 
It is a forward declaration currently.

Reply:
I think it was an include ordering problem due to `DefaultDelete` being in `AK/Types.h`. With the change that @trflynn89 suggested below these includes can be removed. Also could have been something half-baked and I changed those files while trying to get it to work. Either way, not needed now.